inc/saiobject.h

```diff
@@ -145,6 +166,34 @@ sai_status_t sai_bulk_get_attribute(
         _Inout_ sai_attribute_t **attr_list,
         _Inout_ sai_status_t *object_statuses);
 
+/**
+ * @brief Query attribute capability
+ *
+ * @param[in] object_type SAI object type
+ * @param[in] attr_id SAI attribute ID
+ * @param[out] attr_capability Capability per operation
+ *
+ * @return #SAI_STATUS_SUCCESS on success, failure status code on error
+ */
+sai_status_t sai_query_attribute_capability(
+        _In_ sai_object_type_t object_type,
+        _In_ sai_attr_id_t attr_id,
+        _Out_ sai_attr_capability_t *attr_capability);
```

Comment:
Also if libsai will support multiple asic at the same time (multiple switches created) there can be possibility that same attribute is not implemented/not supported on one of those asics, and this api has no way to tell the difference

Reply:
Thanks for the input, I will add switch ID as parameter